networks/http/http.go

```diff
@@ -30,10 +33,19 @@ type Network struct {
 	chainHash string
 	client    client.Client
 	publicKey kyber.Point
+	scheme    crypto.Scheme
 }
 
 // NewNetwork constructs a network for use that will use the http client.
 func NewNetwork(host string, chainHash string) (*Network, error) {
+	if !strings.HasPrefix(host, "http") {
+		host = "https://" + host
```

Comment:
do we want to allow raw `http://` here? I  suppose that's up to the user, but worth thinking about

Reply:
We do 🤷🏻